### PR TITLE
Fix type in class comment

### PR DESCRIPTION
**Pull request by davidschreiber**
This fixes a small typo I discovered while browsing the project's sources.

### CODE REVIEW

**Review by marc-antoine-girard — APPROVED** [review]

**Review by Eclextic — APPROVED** [review]
Tis good

**Comment by davidschreiber** [comment]
Hey folks :) Should we merge this? If it's not desired, we can also close the PR instead. 

@marc-antoine-girard I know it's been some time - thank you for the review! I can't merge this PR myself, so I'm just going to warm it up again as part of housekeeping :D

**Comment by davidschreiber** [comment]
Closing this for the lack of progress.

**Pull request closed without merging**
